index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link  href="style.css" rel="stylesheet">
+   
+    <title>Zuhra Hashimi - Portfolio</title>
+</head>
+
+<body>
+    <div>
+    <section>
+        <header>
+            <div>
+            <nav class="topnav">
+            
+                <a href="#home" class="logo">Zuhra Hashimi</a>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </div>
+        </header>
+    </section></div>
+
+    <section class="main-content">
+        <div class="container">
+            
+            <h1 class="container headline">Hey There.<br>I'm Zuhra</h1>
+            <h2 class="container job-title">I am a Software developer </h2>
+            <p class="container job-description">I can help you build a product,
+                feature or <br>
+                website Look through some of my work<br>
+                and experience! If you like what you see<br>
+                and have a project you need coded,<br>
+                  do not hestiate to contact me.</p>
+        </div>
+    
+        <div class="links">
+            <ul class="social_media">
+                <li>
+                    <a href="https://github.com/ZuhRa-HashiMi" id="git" target="blank"><img src="assets/github.svg" alt="github"></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/zuhra-hashimi-601966214/" id="linkedin" target="blank"><img src="assets/Linkedin.svg" alt="linkedin"></a>
+                </li>
+             
+                <li>
+                    <a href="https://twitter.com/ZuhraHashimi4" id="twitter" target="blank"><img src="assets/twitter.svg" alt="twitter"></a>
```

Comment:
thanks so much for reviewing my code.
I did all of them, if you want to give me some other feedback so please kindly send optional feedback, because this is my extra review request.

Reply:
> thanks so much for reviewing my code. I did all of them, if you want to give me some other feedback so please kindly send optional feedback, because this is my extra review request.

@ZuhRa-HashiMi if you have fixed all suggestions, please, ask for the next review.